nginx.conf

```
@@ -92,6 +92,8 @@ http {
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/developing/assemblyscript-api/$               $scheme://$http_host/docs/$1/developing/graph-ts/api/                       permanent;
         rewrite ^/docs/en/substreams/(?!index\.).+$                                     https://substreams.streamingfast.io                                         permanent;
         rewrite ^/docs/en/firehose/(?!index\.).+$                                       https://firehose.streamingfast.io                                           permanent;
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/deploying/deploying-a-subgraph-to-studio/$    $scheme://$http_host/docs/$1/deploying/deploy-using-subgraph-studio/        permanent;
```

Should we add another redirect from `/deploying/subgraph-studio`, which seems to have been removed as well?
```suggestion
        rewrite ^/docs/([a-zA-Z][a-zA-Z])/deploying/deploying-a-subgraph-to-studio/$    $scheme://$http_host/docs/$1/deploying/deploy-using-subgraph-studio/        permanent;
        rewrite ^/docs/([a-zA-Z][a-zA-Z])/deploying/subgraph-studio/$    $scheme://$http_host/docs/$1/deploying/deploy-using-subgraph-studio/        permanent;
```